WikipediaUnitTests/Code/LocationManagerTests.swift

```diff
@@ -84,6 +93,12 @@ final class LocationManagerTests: XCTestCase {
         XCTAssertEqual(delegate.location, location)  // Test new location on delegate
     }
 
+    func testLastKnowLocation() {
```

Comment:
`testExistingLocationWithoutCallback`
// When the location has already been fetched, a new instance of CLLocationManager contains the last known location in its `location` variable, even before `startMonitoringLocation()` has been called.